Bots/Python/Consumers/CodeFirst/SimpleHostBot/app.py

```diff
@@ -84,6 +84,6 @@ async def messages(req: Request) -> Response:
 
 if __name__ == "__main__":
     try:
-        web.run_app(APP, host="localhost", port=CONFIG.PORT)
+        web.run_app(APP, port=CONFIG.PORT)
```

Comment:
This change doesn't brake anything?

Reply:
It doesn't, using Waterfall to test the Proactive message to see the url and its working perfectly, both locally and in the pipelines.